src/main.rs

```diff
@@ -205,6 +214,10 @@ where
                                 paused.fetch_xor(true, Ordering::SeqCst);
                                 display_handler.unpark();
                             }
+                            Event::Key(Key::Char('\t')) => {
+                                ui_offset.fetch_add(1, Ordering::SeqCst);
+                                display_handler.unpark();
```

Comment:
This causes a bug: try to download something big and then open the app and spam TABs. What would happen is that you'll see the download speed going down. The reason is that we refresh the UI after every tab, and since the display thread is also in charge of measuring the bandwidth (in units of 1 second), it throws the calculation off.

I see why you did this, and of course also realize this happens with the pause too (but because of usability, this is less likely to happen with pause).

We should probably add some sort of "refresh ui without changing state" method. But that can be done separately.